plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveBucketingV1.java

```diff
@@ -40,6 +40,9 @@ private HiveBucketingV1() {}
 
     static int getBucketHashCode(List<TypeInfo> types, Page page, int prefixChannels, int position)
     {
+        checkArgument(types.size() == prefixChannels);
```

Comment:
Seems redundant to check this since we already passed prefixChannels as bucketTypeInfos.size() in HivePartitionedBucketFunction#getBucket